src/site/antora/modules/ROOT/pages/faq.adoc

```diff
@@ -181,25 +181,25 @@ It creates a new message object for each call.
 == How do I log my domain object without creating garbage?
 
 One option is to let the domain object implement `CharSequence`.
-However, for many domain objects it may not be trivial to implement this without allocating temporary objects.
+However, for many domain objects, it may not be trivial to implement this without allocating temporary objects.
 
-An alternative is to implement the `org.apache.logging.log4j.util.StringBuilderFormattable` interface.
-If an object is logged that implements this interface, its `formatTo(StringBuilder)` method is called instead of `toString()`.
+An alternative is implementing the `org.apache.logging.log4j.util.StringBuilderFormattable` interface.
+If an object that implements this interface is logged, its `formatTo(StringBuilder)` method is called instead of `toString().`
 
 [#logger-wrapper]
 == How do I create a custom logger wrapper that shows the correct class, method, and line number?
 
-Log4j remembers the fully qualified class name (FQCN) of the logger and uses this to walk the stack trace for every log event when configured to print location.
+Log4j remembers the logger's fully qualified class name (FQCN) and uses this to walk the stack trace for every log event when configured to print location.
 
 [WARNING]
 ====
 Be aware that logging with location is slow and may impact the performance of your application.
 ====
 
-The problem with custom logger wrappers is that they have a different FQCN than the actual logger, so Log4j can't find the place where your custom logger was called.
+The problem with custom logger wrappers is that they have a different FQCN than the actual logger, so Log4j can't find where your custom logger was called.
 
 The solution is to provide the correct FQCN.
-The easiest way to do this is to let Log4j generate the logger wrapper for you.
+The easiest way is to let Log4j generate the logger wrapper for you.
 Log4j comes with a Logger wrapper generator tool.
 This tool was originally meant to support custom log levels and was moved to the
 {logging-services-url}/log4j/transform/latest/index.html[Log4j Transform subproject].
```

Comment:
The tool was removed from Log4j Transform too, should we remove this paragraph?

Reply:
@ppkarwasz, good catch, yes, we should.